mteb/abstasks/AbsTask.py

```diff
@@ -18,7 +18,11 @@ def load_data(self, **kwargs):
         if self.data_loaded:
             return
 
-        self.dataset = datasets.load_dataset(self.description["hf_hub_name"])  # TODO: add split argument
+        # TODO: add split argument
+        self.dataset = datasets.load_dataset(
+            self.description["hf_hub_name"], 
+            revision=self.description["revision"]
```

Comment:
Should we have a default value for `revision`, or do we make it a required field in all tasks' descriptions? (if the latter, then it's better to add the `revision` field in all the tasks in this same PR)

Reply:
Made it optional for now, but I think we should add it for all datasets.